client/state/email-accounts/schema.js

```diff
@@ -0,0 +1,45 @@
+export const usersSchema = {
+	type: 'array',
+	items: {
+		oneOf: [
+			{
+				type: 'object',
+				required: [
+					'email',
+					'mailbox',
+					'domain',
+					'firstname',
+					'lastname',
+					'fullname',
+					'site_id',
+					'suspended',
+					'product_name',
+					'product_slug',
+				],
+				properties: {
+					mailbox: { type: 'string' },
+					domain: { type: 'string' },
+					email: { type: 'string' },
+					firstname: { type: 'string' },
+					lastname: { type: 'string' },
+					fullname: { type: 'string' },
+					site_id: { type: 'integer' },
+					suspended: { type: 'boolean' },
+					agreed_to_terms: { type: 'boolean' },
```

Comment:
Same thing here with `has_agreed_to_terms`. Note I understand where we are coming from but I see this work on implementing a new email provider as an opportunity to improve a few things. I'm planning to fixing this in the new API layer 🙂.